cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/TimeSeriesCategoryEndpointInput.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2021 Hydrologic Engineering Center
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package mil.army.usace.hec.cwms.radar.client.controllers;
+
+import java.util.Optional;
+import mil.army.usace.hec.cwms.http.client.EndpointInput;
+import mil.army.usace.hec.cwms.http.client.HttpRequestBuilder;
+
+/**
+ *
+ */
+public final class TimeSeriesCategoryEndpointInput extends EndpointInput {
+
+    private final String categoryId;
+    private String officeId;
+
+    public TimeSeriesCategoryEndpointInput() {
+        this.categoryId = null;
+    }
+
+    public TimeSeriesCategoryEndpointInput(String categoryId) {
+        this.categoryId = categoryId;
+    }
+
+    TimeSeriesCategoryEndpointInput officeId(String officeId) {
+        this.officeId = officeId;
+        return this;
+    }
+
+    Optional<String> getCategoryId() {
+        return Optional.ofNullable(categoryId);
+    }
+
+    @Override
+    protected HttpRequestBuilder addInputParameters(HttpRequestBuilder httpRequestBuilder) {
```

Comment:
Where is the category id?

Reply:
It's a URL parameter, not a query parameter (not sure if I have the right terms)

http://localhost:11524/cwms-data/timeseries/category/{category-id}?office=SWT

Reply:
That seems odd... What if the category doesnt exist?

Reply:
Then you get a 404 return code. This is a pretty standard REST design